lochness/redcap/__init__.py

```diff
@@ -52,44 +55,49 @@ def initialize_metadata(Lochness: 'Lochness object',
                         study_name: str,
                         redcap_id_colname: str,
                         redcap_consent_colname: str,
-                        multistudy: bool = True) -> None:
-    '''Initialize metadata.csv by pulling data from REDCap
+                        multistudy: bool = True,
+                        upenn: bool = False) -> None:
+    '''Initialize metadata.csv by pulling data from REDCap for Pronet project
 
     Key arguments:
         Lochness: Lochness object
-        study_name: Name of the study, str.
+        study_name: Name of the study, str. eg) PronetLA
```

Comment:
Where did the `_` go? See Kevin's comment:
https://github.com/AMP-SCZ/lochness/pull/32#discussion_r657208075

Reply:
Great catch. Not sure which comes first, but either I had a mistake in the last comment, or it has been updated since our last discussion to remove `_` in the site name to make it consistent with the `box` & `mediaflux` site directory name. I sort of remember getting a comment to remove `_` from the team. So, to confirm - this is the structure finally shared to teams.

![image](https://user-images.githubusercontent.com/3075783/133630309-b1debb73-55ff-490d-8332-becd13d01095.png)